fastlane_bot/config/constants.py

```diff
@@ -0,0 +1,13 @@
+"""
+This file constains the constants used in the fastlane_bot package.
+
+(c) Copyright Bprotocol foundation 2023.
+Licensed under MIT License.
+"""
+
+FLASHLOAN_FEE_MAP = {
+    # should be in the form of {network: fee * 1e6}
+    "ethereum": 0,
+    "coinbase_base": 0,
+    "fantom": 300,  # Flashloans come with a 0.03% Fee from Beethoven X (Balancer fork)
```

Comment:
Why represent percentage like that?

Reply:
Is this not a standard approach in the industry, to specify in PPM? In any case, just following what we did in the simulator, and as well how Uniswap does it, etc. Happy to change if there is a more preferred method.

Reply:
1. It is not standard, but an implementation-decision (typically per DEX).
2. Unless this is related to something the you need to process before passing as input to a contract function or after receiving as output from a contract function, there is no reason for you to use this resolution to begin with; the only reason for it being used onchain, is the lack of non-integer-arithmetic support, which is obviously not an issue in offchain scripts.

Reply:
In this case, what is your preferred implementation approach / alternative?

Reply:
Why not just use `0.0003` (i.e., multiply by that value wherever needed)?

Reply:
I'd go with the most obvious - either `0.0003 # 0.03%` or `0.03 # %`

Reply:
This has been updated.

Reply:
An important note to make is that Mike's original implementation to use a PPM factor, was actually a good one in the sense of avoiding floating-point calculations as much as possible (i.e., the calculation would subsequently turn into `amount * fee // 1e6`, which is purely integer-arithmetic).
But not critical at this point, especially after he's already changed that.